.patterns

```diff
@@ -1,2 +1,3 @@
 LLDATETIME [A-Za-z]{3}\s+[0-9]{1,2}\s[0-9]{2}:[0-9]{2}
 LLFILESIZE \s*[0-9]+(K|B|G)?
+DISTRIBUTED_TABLE_T_SHARD local='t_s[0-9]+'|agent='[0-255].[0-255].[0-255].[0-255]:[0-9]+:t_s[0-9]+|[0-255].[0-255].[0-255].[0-255]:[0-9]+:t_s[0-9]+'
```

Comment:
This shouldn't be a global CLT pattern since it includes a table name. 
`[0-255].[0-255].[0-255].[0-255]` may be a global pattern. Something containing a table name needs to be made a local pattern.